js_modules/dagster-ui/packages/ui-core/src/app/Util.tsx

```diff
@@ -1,3 +1,4 @@
+import {cache} from 'idb-lru-cache';
```

Comment:
This is an unmaintained library but I took a look at the implementation and the tests and it looks simple enough to me